### ISSUE TITLE
Proteins annotated to DNA-binding transcription factor activity (GO:0003700) should NOT be also annotated to  catalytic activity (GO:0003824)  

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2130]
catalytic activity (GO:0003824)
transcription regulator activity (GO:0140110) 

Probably OK - for eg: 
- Q08211 DHX9 Human: PMID:11416126: RNA helicase and cofactor activity
- Q9H944 MED20 Human: ubi ligase + cofactor activity
Many cofactors are enzymes. 

However 
catalytic activity (GO:0003824) and 
DNA-binding transcription factor activity (GO:0003700) 
give quite a few false-positives. I started to check, in particular human annotations

Annotations (and comments) are here
https://docs.google.com/spreadsheets/d/1pwo5n8kqDW2G2Vnx0s_nh9p5JKFgBGRBTZzX3pnqV18/edit#gid=0

- Note that this report looks different, since it is the 'matrix' report. 
- Note that this is not the full list of annotations, I only included the MODs, since most others are coming from IEA or PAINT. 
- It's not easy to ask for reviews, because this view doesn't allow to tell who has annotated
- I have disputed some annotations in P2GO (mostly human)
- I will though the list and ask for reviews when there are fewer. 

@ValWood can you look at the pombase annotations? there are not very many. 

Thanks, Pascale

**Comment by ValWood** [comment]
I agree, the Pombase ones are all from PAINT. I would not annotate any of these to 
DNA-binding transcription factor activity (GO:0003700)

There is only one exception that I can see that should/could have both
https://www.pombase.org/gene/SPBC1826.01c
TATA-binding protein-associated transcription initiation repressor Mot1

Is a DNA binding protein involved in the regulation of transcription (it binds to the core promoter, so it fits the def).
It is also an ATPase...

**Comment by ukemi** [comment]
PMID:15832388

**Comment by RLovering** [comment]
Hi
I took your gene list (but only the genes with UniProt IDs) and downloaded all the annotations for catalytic activity and dbtf and then have created 2 tabs on your spreadsheet indicating which proteins have been associated with either the catalytic term or the dbTF term by which groups. InterPro annotations can be used to help reduce the number to be checked, but there are a lot still to do. 
I guess the priority would be to check those that InterPro marks as dbTFs but have been curated as catalytic activity, and vice versa

Ruth

**Comment by RLovering** [comment]
Note that InterPro has annotated 3 uniprot records with both dbTFs and catalytic activity

**Comment by ValWood** [comment]
Once the gene products in the intersection are identified  I can make a rule for this.

It would allow 
i) the known catalytic activities  of the enzymes (probably best done with specific GO terms)
ii)  TATA binding protein
iii) Others?

Pascale would it help to remove the PAINT annotations from the spreadsheet so that we see only the primary annotations?

**Comment by RLovering** [comment]
Hi
I am assuming that a protein that binds DNA and regulates transcription should be considered a dbTF
Ruth

**Comment by ValWood** [comment]
Actually, I'm a bit confused.  PomBase only have one annotation (the one above), but lots reported. I assumed they were from PAINT, but they are documented as from PomBase???

**Comment by ValWood** [comment]
I get it. These are coming from PAINT (not sure why it says source PomBase in the spreadsheet)

but in AmiGO:
jmj2 | histone demethylase Jmj2 |   | DNA-binding transcription repressor activity, RNA polymerase II-specific |   | GO_Central | Schizosaccharomyces pombe | IBA | MGI:MGI:104813PANTHER:PTN000075014 | jumonji domain containing protein pthr10694 | protein

I would not annotatate these as 
"DNA-binding transcription repressor activity, RNA polymerase II-specific"

because they don't fit the definition
A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a RNA polymerase II-transcribed gene to repress or decrease transcription. 

the don't bind to specific promoters, they bind to DNA non-specifically to repress chromatin more generally, not at specific genes.

I can see how this could happen if the definition isn't adhered to because this isn't super-obvious from the term name.

**Comment by RLovering** [comment]
Hi
I have gone through all the UCL uniprot annotations, I have marked the changes, I have made or changes that look like they need to be done, on the table and have a couple of questions.

1. For PRKN O60260 see PMID:23985028, is dbTF appropriate?
2. For the TREM family, if it is agreed these are all cofactors can PMID:23077300 be used to annotate MF coactivator for many TRIMs?
3. Why are some enzymes + DNA binding proteins dbTFs and others cofactors? ie how can we capture this in the definition? Could we list protein families that might be misclassified as cofactor or dbTF?

The table you have created only lists one group, even though the annotations have been created by multiple groups. I think doing the pivot table as I have attached as tabs might help identify those groups that need to check their annotations. I don't think you want to suggest all groups look at all of their annotations when many of their annotations are correct. InterPro and this initial review might also help work out the families of proteins and the annotations that need corrections.

Ruth
